.gitignore

```diff
@@ -175,3 +175,5 @@ test/appium/tests/users.py
 ##node bindings
 /bin/
 /lib/
+
+/.calva/
```

Comment:
should this be here? 🤨

Reply:
I don't know if it should be in this PR or added somewhere else, but VS Code and Calva are producing some local output files when connected to the REPL.

Should we add this to .gitignore in another PR, @cammellos / @jakubgs  ?

Reply:
Why another PR? Just do it in a separate commit and don't squash it.